docs/remote-config/overview.mdx

```diff
@@ -3,10 +3,6 @@ title: Remote Config
 sidebar_label: Overview
 ---
 
-:::caution
```

Comment:
please add this caution back in. This plugin is still determined to be beta. See [here](https://firebase.flutter.dev/)

Reply:
In the v0.10.0 changelog, it says
https://pub.dev/packages/firebase_remote_config/changelog

> Graduate package to a stable release. See pre-releases prior to this version for changelog entries.